crates/solver/src/driver.rs

```diff
@@ -30,11 +30,8 @@ use model::{
 use num::{rational::Ratio, BigInt, BigRational, ToPrimitive};
 use primitive_types::{H160, U256};
 use shared::{
-    current_block::{self, CurrentBlockStream},
-    ethrpc::Web3,
-    http_solver::model::SolverRunError,
-    recent_block_cache::Block,
-    tenderly_api::TenderlyApi,
+    current_block::CurrentBlockStream, ethrpc::Web3, http_solver::model::SolverRunError,
+    recent_block_cache::Block, tenderly_api::TenderlyApi,
```

Comment:
I guess cargo format didn't prefer the prettier layout? Kinda like the way it was before.

Reply:
We are all at the mercy of auto-formatting... 😛